lib/RemoteControl/src/App.cpp

```diff
@@ -337,7 +359,13 @@ void App_cmdRspChannelCallback(const uint8_t* payload, const uint8_t payloadSize
     if ((nullptr != payload) && (COMMAND_RESPONSE_CHANNEL_DLC == payloadSize))
     {
         const CommandResponse* cmdRsp = reinterpret_cast<const CommandResponse*>(payload);
-        LOG_DEBUG("CMD_RSP: 0x%02X", cmdRsp->response);
+        LOG_DEBUG("CMD_RSP: ID: 0x%02X , RSP: 0x%02X", cmdRsp->commandId, cmdRsp->responseId);
+        const uint8_t COMMAND_ID_GET_MAX_SPEED = 0x04U; /* Defined here for debugging. */

```

Comment:
If in the future there are more commands, a common header file with the IDs makes sense.